### PR TITLE
[ws-manager-bridge] fix path to entry point

### PR DESCRIPTION
**Pull request by svenefftinge**
## Related Issue(s)

Fixes WEB-608

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
        <li><b>🏷️ Name</b> - se-bridge-fix</li>
        <li><b>🔗 URL</b> - <a href="https://se-bridge-fix.preview.gitpod-dev.com/workspaces" target="_blank">se-bridge-fix.preview.gitpod-dev.com/workspaces</a>.</li>
        <li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
        <li><b>📦 Version</b> - </li>
</ul>

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>

/hold

### CODE REVIEW

File: components/ws-manager-bridge/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "version": "0.1.5",
   "license": "UNLICENSED",
   "scripts": {
-    "start": "node ./dist/src/index.js",
+    "start": "node ./dist/index.js",
```
Comment (svenefftinge) [inline components/ws-manager-bridge/package.json:7]: I don't know why and/or how this worked previously 🤷

Comment (geropl) [reply components/ws-manager-bridge/package.json:7]: Does not start for me.

Comment (geropl) [reply components/ws-manager-bridge/package.json:7]: Fix pushed.

**Review by geropl — APPROVED** [review]
Got a running workspace now. :heavy_check_mark: 

Just waiting for the build to succeed...

**Pull request merged**